Backend/src/main/java/com/onbank/api/validators/AccountNumber.java

```diff
@@ -0,0 +1,15 @@
+package com.onbank.api.validators;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.*;
+
+@Documented
+@Constraint(validatedBy = AccountNumberValidator.class)
+@Target( { ElementType.METHOD, ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AccountNumber {
+    String message() default "Invalid phone number";
```

Comment:
invalid phone number? :D

Reply:
wow, to chyba już efekt zmęczenia wczorajszego :D